src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -514,7 +514,7 @@ public void execute() throws GitException, InterruptedException {
 
     public void clean() throws GitException, InterruptedException {
         reset(true);
-        launchCommand("clean", "-fdx");
+        launchCommand("clean", "-ffdx");
```

Comment:
The last time I considered adding this, I think I found cases that warned me against it while writing tests.  

Could you provide at least one test which confirms that this truly does what you expect it to do?